docs/src/tutorials/child-device-config-mngmnt.md

```diff
@@ -0,0 +1,297 @@
+3. On the right side, under configurations, click on the in *Step 1 Startup / Bootstrap of the child device* created config file (`config1`) in the column: `DEVICE-SUPPORTED CONFIGURATIONS` click on the config file (`config1`)
+4. In the column *AVAILABLE SUPPORTED CONFIGURATIONS* click on the previously added/uploaded configuration snapshot (config1) and click on *Send configuration to device*. This will trigger receiving config snapshot requests via MQTT
+
+The child device agent must subscribe to the `tedge/{child-d}/commands/req/config_update` MQTT topic
+to receive the config update requests from thin-edge.io.
+These requests arrive in the following JSON format:
+
+
+5. Send an “executing” operation status update to acknowledge the receipt of the request via MQTT. On receipt of the request, the agent must send an "executing" MQTT status message as follows:
+
+**Topic:**
+
+`tedge/{child-d}/commands/res/config_update`
+
+**Payload**:
+
+```json
+{
+    "status": "executing",
+    "type": "{config-type}",
+    "path": "/child/local/fs/path"
+}
+```
+
+Example:
+
+`
+mosquitto_pub -h 127.0.0.1 -t "tedge/child1/commands/res/config_snapshot" -m '{"status": "executing", "path": "/home/pi/config/config1", "type": "config1"}'
+`
+
+6. Download the config file update from the URL received in the request via HTTP. After sending this status message, the agent must download the configuration file update
+from the `url` received in the request with an HTTP GET request.
+The agent can then apply the downloaded configuration file update on the device.
+
+Example:
+
+`
+curl http://127.0.0.1:8000/tedge/file-transfer/child1/config_update/config1 --output config1
+`
+
+7. Apply the config file update on the child device
+
+The agent can then apply the downloaded configuration file update on the device.
+
+8. Send a “successful” operation status update via MQTT
+
+Once the update is applied, send a "successful" MQTT status message as follows:
+
+**Topic:**
+
+`tedge/{child-d}/commands/res/config_update`
+
+**Payload**:
+
+```json
+{
+    "status": "successful",
+    "type": "{config-type}",
+    "path": "/child/local/fs/path"
+}
+```
+Example:
+
+`
+mosquitto_pub -h 127.0.0.1 -t "tedge/child1/commands/res/config_snapshot" -m '{"status": "successful", "path": "/home/pi/config/config1", "type": "config1"}'
+`
+
+
+If there are any failures while downloading and applying the update, to remove the entries in Cumulocity IoT, a "failed" status update must be sent instead, to the same topic as follows:
+
+```json
+{
+    "status": "failed",
+    "type": "{config-type}",
+    "path": "/child/local/fs/path"
+}
+```
+
+Example:
+
+`
+mosquitto_pub -h 127.0.0.1 -t "tedge/child1/commands/res/config_snapshot" -m '{"status": "failed", "path": "/home/pi/config/config1", "type": "config1"}'
+`
+
+## References
+
```

Comment:
Provide a reference to the how-to as well.

Reply:
is added